### PR TITLE
web: fix update nickname on settings page

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are fixing deletion wallet when user trying to update his nickname.

### CODE REVIEW

File: web-frontend/src/main.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ import SendCoins from './pages/TransferCoins/SendCoins.tsx';
 import Home from './pages/Home/Home.tsx';
 import ReceiveCoins from './pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx';
 import Settings from './pages/Settings/Settings.tsx';
-import SettingsEdit from './pages/Settings/SeetingsEdit.tsx';
+import SettingsEdit from './pages/Settings/SettingsUpdate.tsx';
```
Comment (mazmassa) [inline web-frontend/src/main.tsx:28]: SettingsEdit -> SettingsEdit ✅ 

now you are changing to:

SettingsEdit -> SettingsUpdate 🚫 

🤯

File: web-frontend/src/models/status.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const BAD_REQUEST = 400;
+export const NOT_FOUND = 404;
+export const UNPROCESSABLE_ENTITY = 422;
+export const INTERNAL_SERVER_ERROR = 500;
```
Comment (mazmassa) [inline web-frontend/src/models/status.ts:4]: 👏

File: web-frontend/src/pages/Settings/SettingsUpdate.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        setError({ message: Intl.t('errors.account-not-found') });
+        break;
+      case UNPROCESSABLE_ENTITY:
+        setError({ message: Intl.t('errors.account-already-exists') });
+        break;
+      default:
+        setError({ message: Intl.t('errors.unexpected-error.description') });
+        break;
+    }
+  }
```
Comment (mazmassa) [inline web-frontend/src/pages/Settings/SettingsUpdate.tsx:59]: Humm, you can't rely all BAD_REQUEST, and/or NOT_FOUND and etc to map to these errors. A BAD_REQUEST and etc can be more things than only invalid format.

File: web-frontend/src/main.tsx (hunk 1 above)
Comment (maxime-carabina) [reply web-frontend/src/main.tsx:28]: Oh sorry I miss that

File: web-frontend/src/pages/Settings/SettingsUpdate.tsx (hunk 3 above)
Comment (maxime-carabina) [reply web-frontend/src/pages/Settings/SettingsUpdate.tsx:59]: > Humm, you can't rely all BAD_REQUEST, and/or NOT_FOUND and etc to map to these errors. A BAD_REQUEST and etc can be more things than only invalid format.

SO what can I do ? 🤔

File: web-frontend/src/main.tsx (hunk 1 above)
Comment (maxime-carabina) [reply web-frontend/src/main.tsx:28]: Done

File: web-frontend/src/pages/Settings/SettingsUpdate.tsx (hunk 3 above)
Comment (maxime-carabina) [reply web-frontend/src/pages/Settings/SettingsUpdate.tsx]: Corrected!
But we have to check if backend return good code (500, 422, etc...) in another PR.

**Pull request merged**